source/fundamentals/crud/write-operations/insert.txt

```diff
@@ -206,10 +212,10 @@ Assume you want to insert the following documents:
 .. code-block:: json
    :copyable: false
 
-   { "_id": 1, "country": "Tanzania" }
-   { "_id": 2, "country": "Lithuania" }
-   { "_id": 1, "country": "Vietnam" }
-   { "_id": 3, "country": "Argentina" }
+   { "_id": 1, "name": "Tanzania" }
+   { "_id": 2, "name": "Lithuania" }
+   { "_id": 1, "name": "Vietnam" }
+   { "_id": 3, "name": "Argentina" }
```

Comment:
Suggestion: It feels a bit jarring to me reading through this to suddenly be talking about locations rather than books, since the rest of the page is using the books collection. Could we maybe adjust this a bit so that it is using something related to books? I don't think it necessarily has to fit into the struct exactly, but it could at least be in the same subject matter.

Reply:
changed to `Book` again with less fields for brevity.